src/styles/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: $star-size;
```

Comment:
The variable `$star-size` is used here, but it's not defined within this file. Ensure that this variable is defined in a file that is imported before this one, or define it here if it's missing.